### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Klatch13** [KenRouKoro/C3H6N6O6#12]
https://pastebin.com/GxZWCtMs

I have another request: please create a discord server, so it will be more convenient to contact you

**Comment by Klatch13** [comment]
https://pastebin.com/kfei84CG

    I removed the Entity and Collections of lithium, as well as the ServerCore mod because someone wrote here that it seemed to be incompatible. And I also removed the DimensionalThreading mod because it was not compatible with C2me. Your mod continues to scribble millions of errors to me anyway. Is this how it should be?

**Comment by KenRouKoro** [comment]
These are not mistakes, you can try to translate it to know what it is.

**Comment by Klatch13** [comment]
It also says Error loading class

**Comment by Klatch13** [comment]
Did I understand correctly that now with this mod the server should not lag when using mega farms?
By the way, ServerCore fixed the incompatibility with your mod.

**Comment by Klatch13** [comment]
https://mclo.gs/3Y91Nno
Please look at error loading class. It is highlighted in orange
@KenRouKoro

**Comment by KenRouKoro** [comment]
Will it come up again after this section?

**Comment by KenRouKoro** [comment]
Theoretically they only appear once, but they appear almost 50 times

**Comment by KenRouKoro** [comment]
These are special compatible Mixin for specific mods and ClassNotFount does not cause any other consequences.

**Comment by KenRouKoro** [comment]
![image](https://user-images.githubusercontent.com/21196863/184820297-3e89d9dd-fd37-4966-8718-1ae2ff963e62.png)
here

**Comment by KenRouKoro** [comment]
Their role is to add the SYNCHRONIZED keyword when loading a specific class.

**Comment by Klatch13** [comment]
That is, it is not necessary to pay attention to it?

**Comment by KenRouKoro** [comment]
Yes, as to why the error is reported, it is because it is reported by the Mixin system and I have no control over it.

**Comment by Klatch13** [comment]
I noticed that you made a MCMT mod. Can it be used with this mod and can you give any recommendation?

**Comment by KenRouKoro** [comment]
Not really, they are completely incompatible. Or rather C3H6N6O6 is completely rewritten to implement MCMT's parallelisation function for entities and is compatible with most content Mods.

**Comment by KenRouKoro** [comment]
Not really, they are completely incompatible. Or rather C3H6N6O6 is completely rewritten to implement MCMT's parallelisation function for entities and is compatible with most content Mods.

**Comment by Klatch13** [comment]
And what is better to use?

**Comment by KenRouKoro** [comment]
MCMTCE is largely incompatible, but works well in pure stock (there are some problems with map generation, but I don't have the energy to fix them).

**Comment by Klatch13** [comment]
Thanks

**Comment by Klatch13** [comment]
And what should be turned off in lithium so that it is compatible with MCMTCE

**Comment by Klatch13** [comment]
Я правильно понял что MCMTCE - это этот мод, только с ещё некоторыми многопоточными вещами? И у него больше несовместимых модов
